mantidproject/fitbenchmarking
# Import unit tests

#### Description of Work:
Added unit tests for functions that import and parse the data and problem definition files.

#### To test:
Run the test files in the mantidpython console and check if the tests are successful (if tests with fail in their name do fail, that is considered expected behaviour).
